.github/workflows/release-please.yml

```
@@ -1,17 +1,35 @@
-# SPDX-FileCopyrightText: 2022 Alliander N.V.
+# SPDX-FileCopyrightText: 2023 Alliander N.V.
 #
 # SPDX-License-Identifier: Apache-2.0
+on:
+  push:
+    branches:
+      - main
 
-name: Release Project
+permissions:
+  contents: write
+  pull-requests: write
 
-on:
-  release:
-    types: [ released ]
+name: release-please
 
 jobs:
+  release_please:
+    runs-on: ubuntu-latest
+    outputs:
+      release_created: ${{ steps.release.outputs.release_created }}
+    steps:
+      - uses: google-github-actions/release-please-action@v3
+        id: release
+        with:
+          release-type: node
```

should be `maven` instead of node
```suggestion
          release-type: maven
```